src/app/component/Header/Header.component.js

```diff
@@ -142,8 +147,7 @@ export class Header extends NavigationAbstract {
             title: true
         },
         [CART_OVERLAY]: {
-            title: true,
-            edit: true
```

Comment:
This triggered to render edit button, but I didn't found where it is used in minicart overlay and it just move header items on desktop.

Reply:
Yes, I believe we deprecated it.